tensorboard/webapp/metrics/views/right_pane/scalar_column_editor/scalar_column_editor_test.ts

```diff
@@ -296,4 +300,20 @@ describe('scalar column editor', () => {
       expect(headerListItems[0].classes['highlight-top']).toBeTrue();
     }));
   });
+
+  describe('closing', () => {
+    it('dispatches metricsSlideoutMenuClosed', () => {
+      const fixture = createComponent();
+      const dispatchedActions: Action[] = [];
+      spyOn(store, 'dispatch').and.callFake((action: Action) => {
```

Comment:
Can this all be setup before-hand, like we typically do in tests?

Reply:
Extracted.